### ISSUE TITLE
[Advice] How much do I break things by importing characters from another server?

### ISSUE THREAD

**Issue by Kuiriel** [ZhengPeiRu21/mod-individual-progression#76]
TL;DR: Do characters need to be 'started' on the server for initial correct flags to be set up, or anything of the sort?

---

My family played the first few levels in the blood elf starting area before I dumped and loaded their characters from my initial azerothcore server to the preview build here.

How much will that have broken the way this works for the rest of the progression system? I imagine it should not be an issue as long as it wasn't done around level 60 etc.

For example, my wife plays warlock, and picked up the imp from her trainer rather than needing to do the old imp summon quest. 

After I moved her across, she doesn't seem to see the old imp summon quest.

I'm not sure if this will mean she will also be able to skip the voidwalker quest, etc.

Sorry if I'm wasting your time with my stupidity.

Thank you

**Comment by StraysFromPath** [comment]
> 

You're best to just import the chars, play for 2 hours, and see what works and what's broken. Nobody can provide an answer better than what you'll find out.

**Comment by ZhengPeiRu21** [comment]
I think you should be fine to use character data from a different AzerothCore installation. There may be some small issue with the old characters but mostly just regarding skipped progression such as learning spells you shouldn't know without completing quests - most issues should sort itself out.

**Comment by ZhengPeiRu21** [comment]
I am going to close this issue as it appears to be resolved. Please feel free to re-open if you need additional help.
